### PR TITLE
Feat/recurring call

### PR DESCRIPTION
**Pull request by tobi-bams**
PR Summary

1. Created the command for adding recurring calls
2. Created the Cron JOB for recurring calls

### CODE REVIEW

**Comment by kevkevinpal** [comment]
closes https://github.com/stakwork/jarvis-backend/issues/103

File: src/builtin/utill/callRecording.ts [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+          attributes: {
+            vars: {
+              media_url: filePathAndName,
+              episode_title: `Jitsi Call on ${dateInUTC}`,
+              clip_description: 'My Clip Description',
+              publish_date: `${dateInUnix}`,
+              episode_image:
+                'https://stakwork-uploads.s3.amazonaws.com/knowledge-graph-joe/jitsi.png',
+              show_img_url:
+                'https://stakwork-uploads.s3.amazonaws.com/knowledge-graph-joe/sphinx-logo.png',
```
Comment (kevkevinpal) [inline src/builtin/utill/callRecording.ts:219]: we might want to use a local image in the future but this works for now

File: src/builtin/utill/callRecording.ts [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+      })) as ChatRecord
+      const filename = extractFileName(recurringCall.link, tribe.jitsiServer)
+      const filepath = formFilenameAndPath(filename, tribe.memeServerLocation)
+
+      const newCall = await fetch(filepath, {
+        method: 'HEAD',
+        headers: { 'Content-Type': 'application/json' },
+      })
+      if (!newCall.ok) {
+        console.log('+++++++++++ No file found yet for', filename)
```
Comment (kevkevinpal) [inline src/builtin/utill/callRecording.ts:99]: we can use sphinx logger here

File: src/builtin/utill/callRecording.ts [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+      }
+      const callRecording: Partial<CallRecordingRecord> = {
+        recordingId: filename,
+        chatId: tribe.id,
+        fileName: `${filename}.mp4`,
+        createdBy: JSON.stringify(createdBy),
+        versionId: callVersionId,
+      }
+      if (!stakwork.ok) {
+        console.log('++++++ Did not save on stakwork')
```
Comment (kevkevinpal) [inline src/builtin/utill/callRecording.ts:132]: we can use sphinxLogger here aswell

**Review by kevkevinpal — APPROVED** [review]
Looks good other than the small comments I made

File: src/builtin/utill/callRecording.ts (hunk 2 above)
Comment (tobi-bams) [reply src/builtin/utill/callRecording.ts:99]: Sure, Lemme update that in my next PR

**Pull request merged**